shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-api/src/main/java/org/apache/shardingsphere/readwritesplitting/api/rule/ReadwriteSplittingDataSourceRuleConfiguration.java

```diff
@@ -64,4 +64,5 @@
     public Optional<String> getReadDataSourceNames() {
         return Optional.ofNullable(props.getProperty("read-data-source-names"));
     }
+
```

Comment:
Please delete the extra blank lines, some indentation changes are not necessary, you can roll back this class.